### PR TITLE
MASTER :: EP-1265 :: Adding energygov facebook followers count to env

### PR DESCRIPTION
**Pull request by chaunceyt**

### CODE REVIEW

File: hourly_follower_count/followers-hourly.py [hunk 1]
```diff
@@ -1,32 +1,33 @@
+import sys
+sys.path.append('/opt/energyapps')
 import csv
 import requests
 import json
+import energy_config
```
Comment (thagler) [inline hourly_follower_count/followers-hourly.py]: I'm curious how python knows to look for this at `/opt/data/glusterfs/energyapps/`, given that the path append a few lines earlier appends `/opt/energyapps`.

**Review by thagler — APPROVED** [review]
I left a question about an `import` statement; that's not a blocker to me approving this PR unless there's a problem with how python handles that import.

**Review by Jwaxo — APPROVED** [review]
Looks good to me.

File: hourly_follower_count/followers-hourly.py (hunk 1 above)
Comment (chaunceyt) [reply hourly_follower_count/followers-hourly.py]: Good catch this is a bug. the /opt/energyapps should be /opt/data/glusterfs/energyapps/.

Comment (chaunceyt) [reply hourly_follower_count/followers-hourly.py]: In the context of the import.

I'm using import sys and then appending /opt/data/glusterfs/energyapps/ to the search path. 
On the glusterfs I have a **energy_config.py** that contains a python dict with the api_token (I didn't want to save that in the repo).
As a result I use energy_config.fb['api_token'] to bring in the token. (this file will be synced to LV)

**Pull request merged**